docs/resources-naming-convention.md

```diff
@@ -0,0 +1,67 @@
+## Resources naming convention
+
+The SDK will be used by external developers, so there is chance to have resources collision [issue](https://github.com/mapbox/mapbox-navigation-android/issues/1793). We add `mapbox`/`Mapbox` prefix to all SDK resource files to avoid resources collision. 
+The below naming convention refers to [ribot's Android Guidelines](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md).
```

Comment:
```suggestion
The SDK is used by external developers, so there is a chance that [resource names could collide](https://github.com/mapbox/mapbox-navigation-android/issues/1793). The Mapbox team has added the `mapbox`/`Mapbox` prefix to all SDK resource files to avoid resource collision. 
The naming convention below refers to [ribot's Android Guidelines](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md).
```